### PR TITLE
Completed Alert and Aavtar Component

### PR DESCRIPTION
**Pull request by varun1233**
1. Completed Home and Documentation Page
2. Completed Alert and Avtar Component

### CODE REVIEW

File: components/avtar/avtar.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
+/* Temp Class */
+.avtar-wrapper {
```
Comment (kaustubh-jsr) [inline components/avtar/avtar.css]: Please correct the avatar spelling. It is correct in your example heading for image and text avatar.

**Review by kaustubh-jsr — COMMENTED** [review]
Apart from that avatar typo, the code structure and preview looks good to me.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.alert {
+  display: flex;
+  align-items: center;
+  min-height: 2rem;
+  border-radius: 0.3rem;
```
Comment (zoef-yt) [inline components/alert/alert.css]: use px

File: components/alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  padding: 0.75rem 1.25rem;
```
Comment (zoef-yt) [inline components/alert/alert.css:6]: use px for units against rem less than 1

File: components/header/header.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
+header {
+  /* background-color: var(--primary-color-light); */
```
Comment (zoef-yt) [inline components/header/header.css]: Remove commented code

**Review by zoef-yt — CHANGES_REQUESTED** [review]
There's some inconsistency between the block of code in CSS make sure you either have space or don't between the code block and make the changes mentioned before.

File: components/header/header.css (hunk 4 above)
Comment (varun1233) [reply components/header/header.css]: Done

File: components/alert/alert.css (hunk 3 above)
Comment (varun1233) [reply components/alert/alert.css:6]: Done

File: components/alert/alert.css (hunk 2 above)
Comment (varun1233) [reply components/alert/alert.css]: Done

**Comment by varun1233** [comment]
> There's some inconsistency between the block of code in CSS make sure you either have space or don't between the code block and make the changes mentioned before.

File: components/avtar/avtar.css (hunk 1 above)
Comment (varun1233) [reply components/avtar/avtar.css]: Done

**Review by zoef-yt — APPROVED** [review]
Good to go.

File: documentation.css [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  font-size: 1.8rem;
+  line-height: 1.5;
+}
+.wrapper .main-documentation .component-intro {
+  margin-bottom: 1.5rem;
+  font-size: 2.2rem;
+  font-weight: 400;
+  margin-top: 1rem;
+}
+.wrapper .main-documentation .component-intro h1 {
```
Comment (Shivani-1524) [inline documentation.css:77]: Try to not chain classes like this due to specificity issues that may arise. Its regarded as bad practice.

File: documentation.css [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+.wrapper .main-documentation .component-usage-explain h3 {
+  font-size: 2.5rem;
+  margin-bottom: 1rem;
+}
+.wrapper .main-documentation .component-display {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 1rem;
+  margin: 2rem 0;
+  align-items: baseline;
```
Comment (Shivani-1524) [inline documentation.css:89]: make sure your component library is consistent. your avatar components are baselined but the text avatar component is center aligned.

**Pull request merged**